randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1095,11 +1132,44 @@ Extra - room_id: [30]
       All of the following:
           Morph Ball
           Any of the following:
+              # Intended path
               Diffusion Missile Data and Missiles
-              # Diagonal Missile shot at the third Missile Block when standing close to the wall can hit it (may not even have to freeze the enemy)
-              Missiles ≥ 3 and Knowledge (Beginner) and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
+              All of the following:
+                  # Diagonal Missile shot at the third Missile Block when being close to the wall can hit it
+                  Missiles ≥ 3 and Knowledge (Advanced)
+                  Any of the following:
+                      # Freeze Ripper and stand on it:
+                      Knowledge (Advanced) and Can Freeze Enemies
```

Comment:
this has a redundant knowledge check here, should be stand on frozen enemies beginner